### DISCUSSION TITLE
question: Multiple cursors + auto complete

### DISCUSSION THREAD

**Discussion by igorluppi** [helix-editor/helix#4405] [Q&A]
Is it possible to perform auto complete while using multiple cursors?
I've tried but the auto complete is performed in the current line, not all lines, forcing me to type everything while using multiple cursors.

**Comment by igorluppi** [comment]
`int something;`
```
some|
some| // here the cursor still on letter e
something| // here auto complete
```

**Comment by the-mikedavis** [comment]
Auto-complete with multiple cursors is not currently implemented: https://github.com/helix-editor/helix/issues/2016
